src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_utils.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.mgmt.servicebus.models.service_bus_management_client_enums import AccessRights
```

Comment:
This is probably why your module is failing load performance.

Reply:
resolved